### DISCUSSION TITLE
Version 7.28.0 compilation error

### DISCUSSION THREAD

**Discussion by maor-sa** [nielsbasjes/yauaa#1560] [Q&A] [answered]
Hi,

I'm attempting to upgrade yauaa from version 7.10.0 to 7.28.0, but I'm encountering a compilation error.

My environment includes Java 11, Spark 3.4.1  and Scala 2.12.5.

Below is the complete build.sbt file:

[build_sbt.txt](https://github.com/user-attachments/files/17016422/build_sbt.txt)
[example.txt](https://github.com/user-attachments/files/17016853/example.1.txt)

**Comment by nielsbasjes** [comment, marked as answer]
Few things:
1. I suggest you create a reproduction project on github. I cannot decipher these text files of yours.
2. How can a compilation error in your code be a problem in yauaa?
3. There is a working scala example included that hasn't changed for a really long time https://github.com/nielsbasjes/yauaa/tree/main/analyzer%2Fsrc%2Fit%2FExamples%2Fscala
